### PR TITLE
test: add action bar e2e tests

### PR DESCRIPTION
**Pull request by Seamoo13**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/4338

#### Please provide a brief summary of this pull request.
add Action bar e2e tests

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by nikvalor — APPROVED** [review]

File: e2e/wdio/platform/pages/action-bar.po.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class ActionBarPo extends BaseComponentPo {
+    url = '/action-bar';
+    root = '#page-content';
+
+    pageTitle = 'app-action-bar-header h1';
+    actionBtnArr = 'fdp-button button';
+    menuItems = 'div fdp-menu-item';
+    backBtnArr = 'fdp-action-bar button[title=\'Go Back\']';
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/platform/pages/action-bar.po.ts:11]: try use 'fdp-action-bar .fd-action-bar__back button' to aviod hardcode Go Back

**Review by AntonOlkhovskyi — CHANGES_REQUESTED** [review]

File: e2e/wdio/platform/pages/action-bar.po.ts (hunk 1 above)
Comment (Seamoo13) [reply e2e/wdio/platform/pages/action-bar.po.ts]: done

File: e2e/wdio/platform/tests/action-bar.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    const actionBarPage = new ActionBarPo();
+
+    beforeAll(() => {
+        actionBarPage.open();
+    }, 1);
+
+    describe('Main checks', function() {
+        it('should check action buttons', ()  => {
+            mouseHoverElement(actionBarPage.actionBtnArr);
+            expect(getCSSPropertyByName(actionBarPage.actionBtnArr, ActionBarContent.hoverColorAttr).value)
```
Comment (valorkin) [inline e2e/wdio/platform/tests/action-bar.e2e-spec.ts:24]: please try to use object destructuring on import, and you will have then
```ts
expect(getCSSPropertyByName(actionBtnArr, hoverColorAttr).value).toContain(saveBtnHvrColor);
```

and code will be much more readable

**Review by valorkin — CHANGES_REQUESTED** [review]
LGTM, but I would love to see e2e more readable please

**Review by AntonOlkhovskyi — APPROVED** [review]

File: e2e/wdio/platform/tests/action-bar.e2e-spec.ts (hunk 2 above)
Comment (Seamoo13) [reply e2e/wdio/platform/tests/action-bar.e2e-spec.ts]: done

**Review by AntonOlkhovskyi — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

File: e2e/wdio/platform/tests/action-bar.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    } = ActionBarContent;
+
+    const {
+        menuItems,
+        actionBtnArr,
+        backBtnArr,
+        descriptions,
+        titles
+    } = new ActionBarPo();
+    const actionBarPage = new ActionBarPo();
```
Comment (valorkin) [inline e2e/wdio/platform/tests/action-bar.e2e-spec.ts:32]: create an object and then destructure it

**Review by valorkin — APPROVED** [review]
LGTM

**Pull request merged**